### PR TITLE
Check for SemanticVersion in the latest pip first

### LINKED ISSUE #292: SemanticVersion no longer works in pip 6.0+
```shell
Traceback (most recent call last):
  File "./run.py", line 658, in <module>
    run()
  File "./run.py", line 639, in run
    print_out_update()
  File "./run.py", line 421, in print_out_update
    SemVer = pip._vendor.distlib.version.SemanticVersion
AttributeError: 'module' object has no attribute 'version'
```

### PR DESCRIPTION
**Pull request by lipis**
Fixes #292

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -418,7 +418,11 @@ def check_for_update():
 
 def print_out_update():
   import pip
-  SemVer = pip.util.version.SemanticVersion
+  try:
+    SemVer = pip._vendor.distlib.version.SemanticVersion
+  except:
```
Comment (gmist) [inline run.py]:
``` python 
except ImportError:
```

Comment (lipis) [inline run.py]: `import pip` is not party of the `try/except` so I'm guessing it will be the `AttributeError`.. no?

Comment (gmist) [inline run.py]: +1 Of course, I was wrong

Comment (lipis) [inline run.py]: Updated and also changed the order, cause in future it will be the opposite :)

**Pull request merged**